t/t-migrate-import.sh

```diff
@@ -1053,6 +1053,25 @@ begin_test "migrate import (copied file)"
 )
 end_test
 
+begin_test "migrate import (copied file with only a single path)"
+(
+  set -e
+
+  setup_local_branch_with_copied_file
+
+  oid="$(calc_oid "$(git cat-file -p :a.txt)")"
+
+  # Prevent MSYS from rewriting /a.txt into a Windows path.
+  MSYS_NO_PATHCONV=1 git lfs migrate import --include="/a.txt" --everything
+
+  # Expect attributes for "/dir/a" and "/a"
```

Comment:
Minor nit: I think this comment should say something like `Expect attribute only for "/a"`.

Reply:
Good point.